docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -16,7 +16,7 @@ Devfiles are YAML files with a defined https://devfile.github.io/devfile/_attach
 
 == Odo and devfile
 
-Odo can now create components from devfiles as recorded in registries. Odo automatically consults the https://github.com/odo-devfiles/registry[default registry] but users can also add their own registries. Devfiles contribute new component types that users can pull to begin development immediately.
+Odo can now create components from devfiles as recorded in registries. Odo automatically consults the https://github.com/odo-devfiles/registry[default registry] but users can also add their own registries. The https://github.com/odo-devfiles/registry[default registry] destn't support s390x arch now. Devfiles contribute new component types that users can pull to begin development immediately.
```

Comment:
should we add a 
`NOTE: xxxx`  here instead of insert one sentence ?

Reply:
@jichenjc Done.